mmaction/apis/inferencers/mmaction2_inferencer.py

```diff
@@ -0,0 +1,228 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+from typing import Dict, List, Optional, Sequence, Tuple, Union
+
+import mmengine
+import numpy as np
+from mmengine.infer import BaseInferencer
+from mmengine.structures import InstanceData
+
+from mmaction.utils import ConfigType
+from .actionrecog_inferencer import ActionRecogInferencer
+
+InstanceList = List[InstanceData]
+InputType = Union[str, np.ndarray]
+InputsType = Union[InputType, Sequence[InputType]]
+PredType = Union[InstanceData, InstanceList]
+ResType = Union[Dict, List[Dict], InstanceData, List[InstanceData]]
+
+
+class MMAction2Inferencer(BaseInferencer):
+    """MMAction2 Inferencer. It's a unified inferencer interface for video
+    analyse task, currently including: ActionRecog. and it can be used to
+    perform end-to-end action recognition inference.
+
+    Args:
+        rec (str, optional): Pretrained action recognition algorithm.
+            It's the path to the config file or the model name defined in
+            metafile. For example, it could be
+            model alias, e.g. ``'slowfast'``,
+            config name, e.g. ``'slowfast_r50_8xb8-8x8x1-256e_kinetics400-rgb'``, # noqa E501
+            or config path. Defaults to None.
```

Comment:
```suggestion
            metafile. For example, it could be:
            
            - model alias, e.g. ``'slowfast'``,
            - config name, e.g. ``'slowfast_r50_8xb8-8x8x1-256e_kinetics400-rgb'``, # noqa E501
            - config path
            
            Defaults to ``None``.
```